plib/hooks/Backup/Selective.php

```diff
@@ -15,12 +15,22 @@ public function getDescription($context)
 
     public function getDatabasesForBackup($context)
     {
-        return $this->_getContextItem($context, 'files');
+        return $this->_getContextItem($context, 'databases');
     }
 
     public function getFilesForBackup($context)
     {
-        return $this->_getContextItem($context, 'databases');
+        $item = $this->_getContextItem($context, 'files');
+
+        if('' === $item){
```

Comment:
please read http://www.php-fig.org/psr/psr-1/ and http://www.php-fig.org/psr/psr-2/